Orange/widgets/utils/owlearnerwidget.py

```diff
@@ -246,8 +246,9 @@ def check_data(self):
         self.Error.sparse_not_supported.clear()
         if self.data is not None and self.learner is not None:
             self.Error.data_error.clear()
-            if not self.learner.check_learner_adequacy(self.data.domain):
-                self.Error.data_error(self.learner.learner_adequacy_err_msg)
+            incompatibility_reason = self.learner.incompatibility_reason(self.data.domain)
```

Comment:
@JakaKokosar, the thing we just spoke about (old learners still using deprecated functions) also applies here.